doc/team_practices.md

```diff
@@ -0,0 +1,36 @@
+# C2 Team Practices
+
+This document outlines best practices for the C2 team, and anyone wanting to contribute to the project.
+
+## Workflow
+
+All code changes should originate in a story on the [story tracker](https://pivotaltracker.com/n/projects/1149728).
+
+Stories are clarified and prioritized in weekly iteration planning meetings (IPM). Team members self-select story ownership.
```

Comment:
Suggestion: `Stories are prioritized with the product owner on a weekly basis. Stories are further clarified, prioritized, and estimated in weekly iteration planning meetings (IPM) which involves the delivery team members and product owner. A story grooming session is held the week before the start of a sprint to discuss implementation and to add more detail into stories before IPM. Team members are encouraged to ask for clarification for stories at any time.`

We are also starting to point stories as a team (at least in this last sprint), which may be worth capturing.